### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by mx-ruthie**
This PR includes correct routing to both my database affirmations table and the external Affirmations API. It also starts the setup for deploying the site and has started disabling the template student data in favor of my own.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
+  "proxy": "http://localhost:8080",
```
Comment (Nmargolis) [inline client/package.json:22]: Why is this necessary? 🤔 we can pair on getting the host as an environment variable or something in the future

File: server/routes/affirmationapi.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const fetch = require("node-fetch");
+//import express from "express";
```
Comment (Nmargolis) [inline server/routes/affirmationapi.js:3]: nit: you can remove this commented out code if you're not using it

File: server/server.js [hunk 3]
```diff
@@ -1,83 +1,91 @@
...
 app.use(cors());
 app.use(express.json());
 
+//needed to install node-fetch@2 in order to do fetch on affirmationsapi.js
+app.use('/random', affirmationApi);
 // creates an endpoint for the route /api
 app.get('/', (req, res) => {
-  res.json({ message: 'Hello from My template ExpressJS' });
+  //res.json({ message: 'Hello from My template ExpressJS' }); Instead of the response being a json it's a 
+  res.sendFile(path.join(REACT_BUILD_DIR, 'index.html'));
```
Comment (Nmargolis) [inline server/server.js:26]: Why does the server need to send an html page?

File: server/server.js [hunk 4]
```diff
@@ -1,83 +1,91 @@
...
-  };
-  console.log([newUser.firstname, newUser.lastname]);
-  const result = await db.query(
-    'INSERT INTO students(firstname, lastname) VALUES($1, $2) RETURNING *',
-    [newUser.firstname, newUser.lastname],
-  );
-  console.log(result.rows[0]);
-  res.json(result.rows[0]);
-});
+// app.post('/api/students', cors(), async (req, res) => {
```
Comment (Nmargolis) [inline server/server.js:45]: It's fine to leave this in for now, if you're going to use it as a reference for how you might add to your affirmations table

**Review by Nmargolis — COMMENTED** [review]
It would be great to update the db.sql file with a dump from your database so that when you set up this project, there's some data in there to work with. ~I followed the steps in the readme to setup, but I think that's what's missing for the server to run for me~ Nevermind, I was on the wrong node version. This would still be nice to include in this or a future PR though

File: server/server.js (hunk 3 above)
Comment (mx-ruthie) [reply server/server.js:26]: I think this has to do with the deployment stuff but pair programming on the second half of that process got punted to next monday.

**Pull request closed without merging**